contents/blog/feature-flags-as-a-service.md

```diff
@@ -0,0 +1,119 @@
+
+## What are feature flags as a service?
+
+A "feature flags as a service" provider is an external application that provides all the functionality for implementing and using feature flags (like [PostHog](/feature-flags)). It is a centralized location to create, manage, evaluate, and monitor your feature flags. This enables remote configuration and the best practice of decoupling deployment from release.
+
+Feature flags as a service integrate with your app like other external services. They have a UI to create flags and then use SDKs or an API to evaluate them. For example, in PostHog, a [feature flag call](/docs/feature-flags/adding-feature-flag-code) is as simple as this:
+
+```python
+from posthog import Posthog
+
+posthog = Posthog('<ph_project_api_key>', host='<ph_instance_address>')
+
+is_my_flag_enabled = posthog.feature_enabled('flag-key', 'distinct_id_of_your_user')
+```
+
+Feature flag services also contain features for more complicated use cases including: 
+
+- Targeting feature flags for percentage rollouts, [betas](/tutorials/public-beta-program), [testing in production](/blog/testing-in-production), and more.
+- Multi-variant flags that enable A/B testing.
+- Integrations with other external services like product analytics, CDPs, and automations.
+- Speed and resiliency optimizations like local evaluation and caching.
+- Logging and analytics.
+
+## Why use a feature flags as a service provider?
+
+The benefit of using a feature flag service is similar to the benefit of using other external services. You get the benefits of feature flags while (theoretically) minimizing their costs. This enables you to focus on building the core functionality that creates value in your business.
+
+> **Aren’t feature flags as a service expensive?** Although you don't pay someone else to build your own feature flag service or use an open source version, you still pay to implement, maintain, and host it. This time, energy, and focus can cost much more than the dollar amount you pay a service provider.
+
+With feature flags as a service, you gain more confidence in:
+
+- **Usability.** The service provides a centralized location and UI to manage flags.
+
+- **Reliability.** The service takes care of the infrastructure, redundancy, and speed.
+
+- **Interoperability.** The service connects to any part of your application (SDKs) and other external services.
+
+In fewer words, you pass your potential problems off to them. 
+
+![Problems](../images/blog/feature-flags-as-a-service/problems.png)
+
+## Deciding whether to build or buy
+
+Like other services, the goal of a feature flag service is to help you build a successful product. Because of this, you should build instead of buy when this is true, such as when your use case is:
```

Comment:
This paragraph is a bit verbose. Can we cut it down a bit?
```suggestion
Ultimately, the decision to build or buy is determined by which approach will enable you to build successful products. Thus, you should build instead of buy when your use is:
```